### PR TITLE
staff

### PR DESCRIPTION
**Pull request by Yu-Yang-Wang**

### CODE REVIEW

File: _staffers/Andy Yang.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+Name: Andy Yang
+Role: Instructor
+Email: Ayang903@bu.edu
+Class Year: 2024
```
Comment (budatasci) [inline _staffers/Andy Yang.md:5]: change to CDS 2024

File: _staffers/Christina Xu.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+Name: Christina Xu
+Role: President
+Email: cjxu@bu.edu
+Class Year: 2023
```
Comment (budatasci) [inline _staffers/Christina Xu.md:5]: change to CAS 2023

File: _staffers/Christina Xu.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+photo: ![F0593E25-2482-4309-AFAE-8D4391BDAAC6_1_105_c](https://user-images.githubusercontent.com/73726129/214954745-0c7bb94e-8a29-421f-9f12-9cf33efbb1d5.jpeg)
+---
+
+I want to provide resources and support to data science and related majors that I wish I had
```
Comment (budatasci) [inline _staffers/Christina Xu.md:9]: delete

File: _staffers/Kewen(Kolvin) Liu.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
+Name: Kewen(Kolvin) Liu
+Role: Vice President
+Email: kewenliu@bu.edu
+Class Year: 2023
```
Comment (budatasci) [inline _staffers/Kewen(Kolvin) Liu.md:5]: change to CAS 2023

File: _staffers/Kewen(Kolvin) Liu.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+photo: ![0X0A7890](https://user-images.githubusercontent.com/73726129/214953553-67d1bd8e-d561-4083-85c5-66571c9ba27a.jpg)
+
+---
+Data science skills are very important in every aspect of work and life and I would love to help people who are interested in gaining such knowledge and skills despite their different backgrounds.
```
Comment (budatasci) [inline _staffers/Kewen(Kolvin) Liu.md:9]: delete

File: _staffers/Lavya Midha.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+---
+Name: Lavya Midha
+Role: Co-Marketing Chair
+Email: lavyam@bu.edu
+Class Year: 2025
```
Comment (budatasci) [inline _staffers/Lavya Midha.md:5]: change to CDS 2025

File: _staffers/Lavya Midha.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+photo: ![q](https://user-images.githubusercontent.com/73726129/214955407-46cf940a-b37a-452a-aa4e-384ef129d0b7.jpeg)
+
+---
+
+Have always been really interested in coding and learning more about data science, this will be a way for me to not just learn about different concepts related to data science but also help other people connect people to similar resources.
```
Comment (budatasci) [inline _staffers/Lavya Midha.md:10]: delete

File: _staffers/Ryan Nie.md [hunk 8]
```diff
@@ -0,0 +1,11 @@
+---
+Name: Ryan Nie
+Role: Tech Team
+Email: ryanznie@bu.edu
+Class Year: 2023
```
Comment (budatasci) [inline _staffers/Ryan Nie.md:5]: change to CAS 2023

File: _staffers/Ryan Nie.md [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+photo: ![IMG_1819](https://user-images.githubusercontent.com/73726129/214955902-9e468248-b5d1-46a1-99cb-fa08121a2479.jpg)
+
+
+---
+DS/ML offers new perspectives at explaining and finding patterns in the world, and being a part of this revolutionary field is very rewarding to me.
```
Comment (budatasci) [inline _staffers/Ryan Nie.md:10]: delete

File: _staffers/Serena Theobald.md [hunk 10]
```diff
@@ -0,0 +1,12 @@
+---
+Name: Serena Theobald
+Role: Tech-team
+Email: serenat@bu.edu
+Class Year: 2025
```
Comment (budatasci) [inline _staffers/Serena Theobald.md:5]: change to CDS 2025

File: _staffers/Serena Theobald.md [hunk 11]
```diff
@@ -0,0 +1,12 @@
...
+photo: ![IMG_7760](https://user-images.githubusercontent.com/73726129/214956269-f09cf523-7df9-4f55-802b-5b7f73da87ec.jpeg)
+
+
+---
+
+ I’d like to help provide data science undergrads with more resources by hosting coding workshops and datathons.
```
Comment (budatasci) [inline _staffers/Serena Theobald.md:11]: delete

File: _staffers/Valerie(Yuyang) Wang.md [hunk 12]
```diff
@@ -0,0 +1,11 @@
+---
+Name: Yuyang Wang
+Role: Tech Team
+Email: uyang68@bu.edu
+Class Year: 2024
```
Comment (budatasci) [inline _staffers/Valerie(Yuyang) Wang.md:5]: change to CAS 2024

File: _staffers/Valerie(Yuyang) Wang.md [hunk 13]
```diff
@@ -0,0 +1,11 @@
...
+photo: ![IMG_2628 (1)](https://user-images.githubusercontent.com/73726129/214960052-1ac8f45d-aa43-4e9a-8b6f-50157608b247.JPG)
+
+
+---
+
+I want to see more practical use of statistical knowledge.
```
Comment (budatasci) [inline _staffers/Valerie(Yuyang) Wang.md:11]: delete

File: _staffers/Veer Sawhney.md [hunk 14]
```diff
@@ -0,0 +1,8 @@
+---
+Name: Veer Sawhney
+Role: Marketing Chair
+Email: vsawhney@bu.edu
+Class Year: 2023
```
Comment (budatasci) [inline _staffers/Veer Sawhney.md:5]: change to CAS 2023

File: _staffers/Vy Nguyen.md [hunk 15]
```diff
@@ -0,0 +1,11 @@
+---
+Name: Vy Nguyen
+Role: Treasurer
+Email: nptv1207@bu.edu
+Class Year: 2024
```
Comment (budatasci) [inline _staffers/Vy Nguyen.md:5]: change to CDS 2024

File: _staffers/Vy Nguyen.md [hunk 16]
```diff
@@ -0,0 +1,11 @@
...
+photo: ![IMG_0024](https://user-images.githubusercontent.com/73726129/214957082-ffc231dd-9731-456a-91f2-a3e966ccca43.JPG)
+
+
+---
+
+ I want to deepen my data science knowledge and being surrounded by people who have similar interest as me
```
Comment (budatasci) [inline _staffers/Vy Nguyen.md:11]: delete

**Review by budatasci — CHANGES_REQUESTED** [review]
looks good so far, just added a couple of minor changes :)

**Comment by Yu-Yang-Wang** [comment]
resolve all request

**Pull request merged**